src/stories/FloatingButton/FloatingButton.stories.ts

```diff
@@ -0,0 +1,58 @@
+import {FloatingButtonControl, meta} from "./FloatingButtonMeta";
+import {Meta, StoryObj} from "@storybook/web-components";
+
+import {html} from "lit";
```

Comment:
Regroup external import at the top and local import at the bottom with a line separation
-> Just move up line 4 content to line 3